hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1464,6 +1464,12 @@
 
   public static final String HBASE_CANARY_READ_RAW_SCAN_KEY = "hbase.canary.read.raw.enabled";
 
+  public static final String HBASE_CANARY_INFO_PORT = "hbase.canary.info.port";
```

Comment:
Is it possible to not add these configurations in HConstants? Just put them in the CanaryTool class?

Reply:
> Is it possible to not add these configurations in HConstants? Just put them in the CanaryTool class?

ok, I will modify the code here